src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { PUBLIC_IMAGE_FOLDER, DEFAULT_BANNER_IMAGE, total } from '../../config/constants';
+import Img from './style';
+import { getRandomNumber, getNextRoundRobin } from '../../libs/utils/math';
+
+class Slider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      current: -1,
+
+    };
+  }
+
+  componentDidMount = () => {
+    const { random, duration } = this.props;
+    let { current } = this.state;
+    this.id = setInterval(() => {
+      if (random) {
+        current = getRandomNumber(total);
+      } else {
+        current = getNextRoundRobin(current, total);
+      }
+      this.setState({ current });
+    }, duration);
+  }
+
+  componentWillUnmount = () => {
+    clearInterval(this.id);
+  }
+
+  render() {
+    const { current } = this.state;
+    const {
+      altText, height, duration, banner,
+    } = this.props;
+    const { defaultbanner } = this.props;
```

Comment:
destructure defaultbanner along with altTest and others. No need to destructure twice from props. Use camelCase for defaultbanner as it is also given ticket

Reply:
destructured the defaultBanner along with altTest and others. Used camelCase for defaultBanner